opcodes/arc-dis.c

```diff
@@ -117,6 +117,11 @@ typedef struct skipclass
    disassembling.  */
 static linkclass decodelist = NULL;
 
+/* ISA mask value enforced via disassembler info options.  ARC_OPCODE_NONE
+   value means that no CPU is enforced.  */
+
+static unsigned enforced_isa_mask = ARC_OPCODE_NONE;
```

Comment:
What would be the use case for this? If option is not specified at all, then disassembler revert to current behavior - checking ELF headers. If instead it will use CPU value that was specified with --with-cpu, then that will change how `objdump -d` works - it will always print disasm for default CPU, ignoring the value in ELF file.